### ISSUE TITLE
Battle Items

### ISSUE THREAD

**Issue by RiftSpatial** [timinc-cobble/cobblemon-unimplementeditems#12]
Such as Toxic Orb, Flame Orb, Life Orb.

**Comment by tmetcalfe89** [comment]
help wanted: how do I make a held item that's recognized by the battle system?

**Comment by RiftSpatial** [comment]
I noticed that evidently you can apply the effects from an item in the showdown folder to any item and it may just work? Just something I read in the dev server, probably worth checking it out, unless you already knew about it.

**Comment by tmetcalfe89** [comment]
> I noticed that evidently you can apply the effects from an item in the showdown folder to any item and it may just work? Just something I read in the dev server, probably worth checking it out, unless you already knew about it.

Yep, that's what I'd like to do. Short of a code example, this is just repeating what I would like to do though. Adding items has been paused until 1.4 comes out, but I may get the speed cutting (for EV boosters) in there before then.
